src/components/ChatLog.jsx

```diff
@@ -0,0 +1,45 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = ({entries, handleLike, currentUser}) => {
+    const getChatLog = (entries) => {
+    return entries.map((entry) => {
+    return (
+        <ChatEntry                                 
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            handleLike={handleLike}
+            currentUser={currentUser}
+        />
+    )
+})
+};
```

Comment:
Be on the lookout for proper indentation and semi-colons.
```suggestion
    const getChatLog = (entries) => {
      return entries.map((entry) => {
        return (
          <ChatEntry                                 
            key={entry.id}
            id={entry.id}
            sender={entry.sender}
            body={entry.body}
            timeStamp={entry.timeStamp}
            liked={entry.liked}
            handleLike={handleLike}
            currentUser={currentUser}
          />
        );
      });
    };
```